File: LeetCode/1768-merge-strings-alternately/1768-merge-strings-alternately.js
var mergeAlternately = function(word1, word2) {
  let answer = '';
  for (let i = 0; i < Math.max(word1.length, word2.length); i++) {
    if (i < word1.length) {
      answer += word1[i];
    }
    if (i < word2.length) {
      answer += word2[i];
    }
  }
  return answer;
};

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        one = len(word1)
        two = len(word2)

        smallest = min(one, two)

        mergeResponse = ''

        for letter in range(smallest):
            mergeResponse += word1[letter]
            mergeResponse += word2[letter]

        if one < two:
            mergeResponse += word2[one: ]
        else: 
            mergeResponse += word1[two: ]
        
        return mergeResponse
        